src/common/guards/permission/permission.guard.ts

```
@@ -1,11 +1,11 @@
 import {
-  CanActivate,
-  ExecutionContext,
-  ForbiddenException,
+    CanActivate,
```

ESLint

[error] Delete `··`

(prettier/prettier)